os-config.tpl.yml

```diff
@@ -146,6 +146,7 @@ rancher:
       - /usr/bin/ros:/usr/bin/cloud-init-save:ro
       - /usr/bin/ros:/usr/sbin/netconf:ro
       - /usr/bin/ros:/usr/sbin/wait-for-docker:ro
+      - /usr/rancher:/usr/share/rancher:ro
```

Comment:
I put them in different directories because when they were the same, they sometimes over-wrote each other and ended up empty :(